components/sortable-grid.js

```diff
@@ -184,13 +184,11 @@ function HeaderCell({ cells, cellIndex, meta, children }) {
   // header cell title renderers. A child of type "object" indicates a React component, while
```

Comment:
I feel that you may need to update the comment here since you don't distinguish whether the child is object or not any more?